File: java/mohit/Trees-BST/SearchInBST.java
static TreeNode solve(TreeNode root, int target)
{
    //CODE HERE
    if(root == null) return new TreeNode(-1);
    if(root. val == target) return root;
    if(target < root.val) return solve(root.left, target);
    else return solve(root.right, targer);
}

File: Python/vedant_bhatnagar/DevsnestProblems_THA/Phase_2/Trees/searchInBST.py
def solve(root, target):
    # CODE HERE
    if not root:
        return TreeNode(-1)
    ptr = root    
    while ptr != None:
        if ptr.val == target:
            return ptr
        if target > ptr.val:
            ptr = ptr.right
        else:
            ptr = ptr.left
    return TreeNode(-1)